src/main/java/uk/gov/justice/digital/service/DomainRefreshService.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.justice.digital.service;
+
+import org.apache.spark.sql.SparkSession;
+import uk.gov.justice.digital.client.dynamodb.DynamoDBClient;
+import uk.gov.justice.digital.domains.DomainExecutor;
+import uk.gov.justice.digital.domains.model.DomainDefinition;
+import uk.gov.justice.digital.repository.DomainRepository;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Set;
+import java.util.logging.Logger;
+
+// TODO Rename it to DomainService??
+public class DomainRefreshService {
+
+    protected String domainFilesPath;
+    protected String domainRepositoryPath;
+    protected String sourcePath;
+    protected String targetPath;
+    protected DomainRepository repo;
+
+    static Logger logger = Logger.getLogger(DomainRefreshService.class.getName());
+
+    public DomainRefreshService(final SparkSession spark,
+                                final String sourcePath,
+                                final String targetPath,
+                                final DynamoDBClient dynamoDBClient) {
+        this.sourcePath = sourcePath;
+        this.targetPath = targetPath;
+        this.repo = new DomainRepository(spark, dynamoDBClient);
+    }
+
+    public void run(final String domainTableName, final String domainId, final String domainOperation) {
+        Set<DomainDefinition> domains = getDomains(domainTableName, domainId);
+        logger.info("Located " + domains.size() + " domains for name '" + domainId + "'");
+        for(final DomainDefinition domain : domains) {
+            processDomain(domain, domainOperation);
+        }
+    }
+
+    protected Set<DomainDefinition> getDomains(final String domainTableName, final String domainId) {
+        return this.repo.getForName(domainTableName, domainId);
+    }
+
+    protected void processDomain(final DomainDefinition domain, final String domainOperation) {
+        try {
+            logger.info("DomainRefresh::process('" + domain.getName() + "') started");
+            final DomainExecutor executor = new DomainExecutor(sourcePath, targetPath, domain);
+            executor.doFull(domainOperation);
+            logger.info("DomainRefresh::process('" + domain.getName() + "') completed");
```

Comment:
No need to include the source class or method name. This is part of the log message so can be simplified.

This applies to any other log message where the class and method name are included.